### DISCUSSION TITLE
Overriding layout for root route

### DISCUSSION THREAD

**Discussion by javadude24** [sveltejs/kit#2999] [Q&A]
### Describe the bug

I’m building a site, using sveltekit and plan to have a home page (via index.svelte) route with no layout. However, all other pages will have a common layout, so I plan to define __layout.svelte in routes directory. How do I reset the layout for index.svelte. I tried to define a directory ‘index’ with index.svelte and __layout.reset.svelte files within, but that does not work. Please advise a solution. 

### Reproduction

Yes

### Logs

```shell
N/A
```

### System Info

```shell
N/A
```

### Severity

blocking all usage of SvelteKit

### Additional Information

Need a solutiuon

**Comment by mihaon** [comment]
It would be great to add an index.reset.svelte feature to svelte kit. Related issue: https://github.com/sveltejs/kit/issues/2694

Without this feature, the only way I can see is:

**_routes/\_\_layout.svelte_**
```
<script>
	import { page } from '$app/stores'
</script>

{#if $page.path == '/'}
	<slot></slot>
{:else}
	<header>Layout header</header>
	<main>
		<slot></slot>
	</main>
	<footer>Layout footer</footer>
{/if}
```

**_routes/index.svelte_**
```
<p>Content of index.svelte</p>
```

**_routes/about.svelte_**
```
<p>Content of about.svelte</p>
```

### Results

_**http://localhost:3000/**_
```
<p>Content of index.svelte</p>
```

_**http://localhost:3000/about**_
```
<header>Layout header</header>
<main>
	<p>Content of about.svelte</p>
</main>
<footer>Layout footer</footer>
```
